examples/stest.py

```diff
@@ -66,10 +66,19 @@ def run_test(command, expected={}, verbose=False):
 
     output = readjson.json_parse_lines(stdout.splitlines())
 
-    for key, expected_val in expected.items():
+    for key, expected_valpair in expected.items():
         test_val = output[key]
 
-        if abs(float(expected_val) - float(test_val)) > test_tol:
+        try:
+            expected_val = expected_valpair[0]
+            test_tol = expected_valpair[1]
+        except TypeError:
```

Comment:
If I understand correctly, here you use exception as "`if`": if expected_valpair has size 2, then do the `try` part, otherwise do the other part. This works fine, but I am wondering if we can use a normal "`if`" (the exception is not really an "error").